tests/Aspire.Hosting.Docker.Tests/Snapshots/DockerComposePublisherTests.PublishAsync_GeneratesValidDockerComposeFile.verified.yaml

```diff
@@ -16,6 +16,15 @@
       - "8001:8000"
     networks:
       - "aspire"
+    configs:
+      - source: "cache__usr_local_share_redis.conf"
+        target: "/usr/local/share/redis.conf"
+        mode: 0644
+      - source: "cache__usr_local_share_folder_file.sh"
+        target: "/usr/local/share/folder/file.sh"
+        uid: 1000
+        gid: 1000
+        mode: 0700
```

Comment:
This is *VERY* cool 😄

Reply:
I was definitely happy to find out compose configs can support basically the full set of configuration from WithContainerFiles. Makes having a consistent runtime and publish experience a lot easier.

Reply:
Spoke too soon; there is one headache in that compose will ignore uid, gid, and mode when using source file mode (vs specifying inline content). They use a volume when a specific file is referenced, but `cp` when contents are provided.

One option would be to simply read text and convert every referenced file to the inline content form, but that wouldn't work if a user was expecting to use WithContainerFiles for a binary file.